cmd/access/node_builder/unstaked_access_node_builder.go

```diff
@@ -312,6 +313,30 @@ func (builder *UnstakedAccessNodeBuilder) enqueueUnstakedNetworkInit() {
 func (builder *UnstakedAccessNodeBuilder) enqueueConnectWithStakedAN() {
 	builder.Component("upstream connector", func(_ *cmd.NodeConfig) (module.ReadyDoneAware, error) {
 		return newUpstreamConnector(builder.bootstrapIdentities, builder.LibP2PNode, builder.Logger), nil
+	}).Component("RPC engine", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
```

Comment:
this file should no longer exist after the refactor.

Reply:
Indeed. I will add a changelist.

Reply:
This is the final layout: https://github.com/GetElastech/flow-go/commit/33623bb268ecfaedd1680372860ab98e6a35b7b6
```
cmd/access/
cmd/access/main.go
cmd/access/README.md
cmd/access/.gitignore
cmd/access/node_builder
cmd/access/node_builder/access_node_builder.go```